assets/js/standings_chart.js

```diff
@@ -0,0 +1,28 @@
+import Chart from "chart.js"
+
+let StandingsChart = {
+  buildChart() {
+    var ctx = document.getElementById('standings-chart').getContext('2d');
+    var myChart = new Chart(ctx, {
+      type: 'line',
+      data: {
+        labels: ['M', 'T', 'W', 'T', 'F', 'S', 'S'],
+        datasets: [
+          {
+            label: 'apples',
+            data: [12, 19, 3, 17, 6, 3, 7],
+            borderColor: "rgba(153,255,51,0.4)",
+            fill: 'false'
+          }, {
+            label: 'oranges',
+            data: [2, 29, 5, 5, 2, 3, 10],
+            borderColor: "rgba(255,153,0,0.4)",
+            fill: 'false'
+          },
```

Comment:
Unexpected trailing comma.

[Read more about it here.](http://eslint.org/docs/rules/comma-dangle)